gateway/1.9/modules/ROOT/pages/policies-included-stream-idle-timeout.adoc

```diff
@@ -0,0 +1,65 @@
+= Stream Idle Timeout Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: basic authentication, api gateway, flex gateway, gateway, policy
+
+[width="100%", cols="5,15"]
+|===
+>s| Policy Name | Stream Idle Timeout
+>s|Summary  | Sets a maximum duration stream can remain idle without receiving a request or response
+>s|Category | Quality of Service
+>s|First Flex Gateway version available | v1.9.0
+^s| Returned Status Codes | 504 - Stream timeout
+|===
+
+== Summary
+
+The Stream Idle Timeout policy enables sets a maximum duration a stream can remain idle without receiving any data in either direction either the inbound (client to Flex Gateway) or outbound (Flex Gateway to upstream service) direction. This policy prevents idle connections from consuming resources unnecessarily. The timeout time starts at the last activity (request or response) on the stream. 
+
+By default, all Flex Gateway API instances have a stream timeout of 300 seconds. Apply the policy overrides this default.
+
+== Configuring Policy Parameters
+
+=== Flex Gateway Local Mode
+
+In Flex Gateway Local Mode, you apply the policy to your API via declarative configuration files. Refer to the following policy definition and table of parameters:
```

Comment:
I recommend going through the rest of the PR and changing "the following" to "the" or "this" or another appropriate word.

Reply:
Hey Tasha, I agree with this and noticed it when I wrote it. We do however use this in all the other policies. We have a policy audit coming post Flex Release, would it be ok if I just included these fixes there?

Reply:
Sure, that makes sense!